src/apis.php

```diff
@@ -418,6 +409,32 @@ protected function cron()
     }
 
     /**
+     * @param array $cronJobs
+     *
+     * @return bool
+     */
+    protected function containsOnlyAllowedCronJobs(array $cronJobs)
+    {
+        $validCronJobs = true;
+
+        if (empty($cronJobs)) {
```

Comment:
this is unnecessary, as the following logic will lead to the same return value

Reply:
you are right, fixed